### PR TITLE
Feat/new addons table

### PR DESCRIPTION
**Pull request by FredrikOseberg**
Implements new addons table

### CODE REVIEW

**Comment by FredrikOseberg** [comment]
[Frontend Tables: Addons [Only table header]](https://trello.com/c/xYRXzUPp/1185-frontend-tables-addons-only-table-header)

File: src/component/addons/AddonList/AvailableAddons/AvailableAddons.tsx [hunk 1]
```diff
@@ -21,40 +32,174 @@ interface IProvider {
...
+            name,
+            displayName,
+            description,
+        }));
+    }, [providers, loading]);
+
+    const columns = useMemo(
+        () => [
+            {
+                id: 'Icon',
```
Comment (olav) [inline src/component/addons/AddonList/AvailableAddons/AvailableAddons.tsx:63]: Maybe `IconCell` would work here.

File: src/component/addons/AddonList/AvailableAddons/AvailableAddons.tsx [hunk 2]
```diff
@@ -21,40 +32,174 @@ interface IProvider {
...
+                    );
+                },
+                sortType: 'alphanumeric',
+            },
+            {
+                Header: 'Actions',
+                id: 'Actions',
+                align: 'center',
+                Cell: ({ row: { original } }: any) => (
+                    <Box
```
Comment (olav) [inline src/component/addons/AddonList/AvailableAddons/AvailableAddons.tsx:106]: Would `ActionCell` work here? Could also extract a `CreateAddonButton` component for readability.

File: src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.tsx [hunk 3]
```diff
@@ -43,27 +38,156 @@ export const ConfiguredAddons = ({ getAddonIcon }: IConfigureAddonsProps) => {
...
+                        />
+                    );
+                },
+                sortType: 'alphanumeric',
+            },
+            {
+                Header: 'Actions',
+                id: 'Actions',
+                align: 'center',
+                Cell: ({ row: { original } }: any) => (
```
Comment (olav) [inline src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.tsx:116]: Could extract an `AddonActionCell` component here as well.

**Review by olav — APPROVED** [review]

**Pull request merged**